docs/man/git-lfs-config.5.ronn

```diff
@@ -8,6 +8,11 @@ precedence. Most options pertaining to git-lfs are contained in the `[lfs]`
 section, meaning they all named `lfs.foo` or similar, although occasionally an
 lfs option can be scoped inside the configuration for a remote.
 
+These settings are read from a file called `.lfsconfig` found at the root of the
+repository, though legacy support exists to read from `.gitconfig` as well.
+
+The `.lfsconfig` file uses the same format as `.gitconfig`.
```

Comment:
How would you word this? Something along these lines?

> These settings are read from a file called `.lfsconfig` found at the root of the repository, though legacy support exists to read from the Git configuration as returned by `git config -l` as well.

And should I open a new PR for that or edit on top of this branch?